timelock-impl/src/main/java/com/palantir/atlasdb/timelock/paxos/AutobatchingPingableLeaderFactory.java

```diff
@@ -64,24 +61,17 @@ public static AutobatchingPingableLeaderFactory create(
             Map<BatchPingableLeader, ExecutorService> executors,
             Duration leaderPingResponseWait,
             UUID localUuid) {
-        BiMap<BatchPingableLeader, ClientAwarePingableLeaderImpl> correspondence = KeyedStream.of(
-                executors.keySet())
-                .map(ClientAwarePingableLeaderImpl::create)
-                .collectTo(HashBiMap::create);
-
-        Map<ClientAwarePingableLeader, ExecutorService> clientAwarePingableExecutors =
-                KeyedStream.stream(correspondence.inverse())
-                        .<ClientAwarePingableLeader>mapKeys(Function.identity())
-                        .map(executors::get)
-                        .collectToMap();
-
-        DisruptorAutobatcher<UUID, Optional<ClientAwarePingableLeader>> uuidAutobatcher =
-                Autobatchers.independent(new GetSuspectedLeaderWithUuid(clientAwarePingableExecutors, localUuid, leaderPingResponseWait))
+        Map<ClientAwarePingableLeader, ExecutorService> clientAwarePingableExecutors = KeyedStream.stream(executors)
+                .<ClientAwarePingableLeader>mapKeys(ClientAwarePingableLeaderImpl::create)
+                .collectToMap();
```

Comment:
yeah, this reads easier than the `BiMap` stuff

Reply:
haha yeah, I was reading it again and I had no idea what was going on 😅